.github/ISSUE_TEMPLATE/1.bug.yml

```diff
@@ -34,17 +34,17 @@ body:
     attributes:
       label: Steps to reproduce 🕹
       description: |
-        Provide a link to a live example (you can use codesandbox.io) and an unambiguous set of steps to reproduce this bug.
-        Include code to reproduce, if relevant (which it most likely is).
+        Please provide a link to a live example and an unambiguous set of steps to reproduce this bug.
+             
+        As a starting point, we recommend you browse our [documentation](https://mui.com/x/advanced-components/), and [select](https://mui.com/static/docs/forking-an-example.png) the closest example to your use case.
 
-        You can use the official codesandbox template of the [DataGrid](https://codesandbox.io/s/datagrid-v5-quick-start-54iz1) or [DataGridPro](https://codesandbox.io/s/datagridpro-v5-quick-start-5pjhc) as a starting point.
-
-        If you have an issue concerning TypeScript please start from this TypeScript playground: https://mui.com/r/ts-issue-template.
+        Or you can use a [basic template](https://mui.com/r/x-issue-template) to build your reproduction case.
```

Comment:
Would it feel better without the blank line before "Or you can use"?

<img width="714" alt="Screenshot 2022-07-09 at 11 45 54" src="https://user-images.githubusercontent.com/3165635/178100544-b35e30a8-8843-47a3-8efc-e08a3e1caaec.png">